### ISSUE TITLE
你好，想问一下，这里不是利用了原始标签进行训练的吗，为什么还是无监督学习呢。论文中提出是划分Vb，所以没有太看明白，想请教一下

### ISSUE THREAD

**Issue by angle2001** [GraphBerry/DiG-In-GNN#1]
>     def get_contrast_sample(self, gen_feats, raw_feats, env_feats, env_raw_feats, labels):
        classify_label_1 = []
        classify_label_0 = []
        for i in range(len(labels)):
            if labels[i] == torch.tensor(1):
                classify_label_1.append(i)
            else:
                classify_label_0.append(i)

**Comment by XZJIsme** [comment]
你好，本文不是无监督学习，对比学习的时候使用了训练集的标签； $\mathcal{V}_b$ 是一个 batch，本文是 mini-batch，请问具体的疑惑是？
